test/advi_locscale.jl

```diff
@@ -0,0 +1,86 @@
+
+const PROGRESS = length(ARGS) > 0 && ARGS[1] == "--progress" ? true : false
+
+using Test
+
+@testset "advi" begin
+    @testset "locscale" begin
+        @testset "$(modelname) $(objname) $(realtype) $(adbackname)"  for
+            realtype ∈ [Float64], # Currently only tested against Float64
+            (modelname, modelconstr) ∈ Dict(
+                :NormalLogNormalMeanField => normallognormal_meanfield,
+            ),
+            (objname, objective) ∈ Dict(
+                :ADVIClosedFormEntropy  => ADVI(10),
+                :ADVIStickingTheLanding => ADVI(10, entropy = StickingTheLandingEntropy()),
+            ),
+            (adbackname, adbackend) ∈ Dict(
+                :ForwarDiff  => AutoForwardDiff(),
+                # :ReverseDiff => AutoReverseDiff(),
+                # :Zygote      => AutoZygote(), 
+                # :Enzyme      => AutoEnzyme(),
+            )
+
+            seed = (0x38bef07cf9cc549d)
+            rng  = StableRNG(seed)
+
+            modelstats = modelconstr(realtype; rng)
+            @unpack model, μ_true, L_true, n_dims, is_meanfield = modelstats
+
+            T, η = is_meanfield ? (5_000, 1e-2) : (30_000, 1e-3)
+
+            b    = Bijectors.bijector(model)
+            b⁻¹  = inverse(b)
+            μ₀   = zeros(realtype, n_dims)
+            L₀   = Diagonal(ones(realtype, n_dims))
```

Comment:
```suggestion
            μ₀   = Zeros(realtype, n_dims)
            L₀   = Diagonal(Ones(realtype, n_dims))
```
from FillArrays (already depending on it because of Distributions.jl)

Reply:
Oh that's right.